src/core/doe_commercial_reference_building_loads.jl

```diff
@@ -134,25 +134,32 @@ function built_in_load(
 
     profile_path = joinpath(lib_path, string("crb8760_norm_" * city * "_" * buildingtype * ".dat"))
     input_normalized = false
+    shift_possible = false
     if isempty(normalized_profile)
         if occursin("FlatLoad", buildingtype)
             normalized_profile = custom_normalized_flatload(buildingtype, year)
         else 
             normalized_profile = vec(readdlm(profile_path, '\n', Float64, '\n'))
+            shift_possible = true
         end
     else
         input_normalized = true
     end
 
+    # The normalized_profile for CRBs (not FlatLoads) is based on year 2017 which starts on a Sunday. 
+    # If the year is not 2017 and we're using a CRB, we need to shift the profile to make the first day of the profile equal to the first day of the input year.
```

Comment:
@Bill-Becker It's possible I'm wrong on this, but I think this needs a slightly different formulation. 

Currently: 
EX: Year = 2021. Start day = Fri (5)
        first_day_of_year = 5
        shift_days = 5- 7 = -2
        normalized_profile = circshift(normalized_profile, 24 * -2)
        ^^ Shifts the first two days (Sun Mon) to the end (moves vector left). So now we are starting on Tues (rather than Friday)

I think this should instead be: 

```
shift_days = (7 - first_day_of_year) % 7 # mod 7 makes it so if first_day_of_year = 7 this is 0
```